Source/Core/Core/PowerPC/PPCSymbolDB.cpp

```diff
@@ -77,6 +77,11 @@ void PPCSymbolDB::AddKnownSymbol(u32 startAddr, u32 size, const std::string& nam
     {
       PPCAnalyst::AnalyzeFunction(startAddr, tf, size);
       checksumToFunction[tf.hash] = &(functions[startAddr]);
+
+      tf.function_name = tf.name.substr(0, tf.name.find('('));
```

Comment:
I see, I have made GetStrippedFunctionName local to PPCSymbolDB and removed the const qualifier.